packages/process/process_monitor.pony

```diff
@@ -459,12 +477,22 @@ actor ProcessMonitor
     _expect = _notifier.expect(this, qty)
     _read_buf_size()
 
-  fun _kill_child() ? =>
+  fun _kill_child() =>
     """
     Terminate the child process.
```

Comment:
I would love to know that this method uses SIGTERM first and, if this fails, SIGKILL from the documentation.

Reply:
Done.